src/tribler/core/components/libtorrent/tests/test_download_manager.py

```diff
@@ -22,24 +24,26 @@ def create_fake_download_and_state():
     tdef = TorrentDef()
     tdef.get_infohash = lambda: b'aaaa'
     fake_peer = {'extended_version': 'Tribler', 'id': 'a' * 20, 'dtotal': 10 * 1024 * 1024}
-    fake_download = MagicMock()
-    fake_download.get_def = lambda: tdef
-    fake_download.get_def().get_name_as_unicode = lambda: "test.iso"
-    fake_download.get_peerlist = lambda: [fake_peer]
-    fake_download.hidden = False
-    fake_download.checkpoint = lambda: succeed(None)
-    fake_download.stop = lambda: succeed(None)
-    fake_download.shutdown = lambda: succeed(None)
-    dl_state = MagicMock()
-    dl_state.get_infohash = lambda: b'aaaa'
-    dl_state.get_status = lambda: DownloadStatus.SEEDING
-    dl_state.get_download = lambda: fake_download
-    fake_config = MagicMock()
-    fake_config.get_hops = lambda: 0
-    fake_config.get_safe_seeding = lambda: True
-    fake_download.config = fake_config
-
-    return fake_download, dl_state
+    fake_download = MagicMock(
+        get_def=MagicMock(return_value=tdef),
+        get_peerlist=MagicMock(return_value=[fake_peer]),
+        hidden=False,
+        checkpoint=AsyncMock(),
+        stop=AsyncMock(),
+        shutdown=AsyncMock(),
+        config = MagicMock(
+           get_hops=MagicMock(return_value=0),
+            get_safe_seeding = MagicMock(return_value=True),
+        ),
+    )
+    fake_download.get_def().get_name_as_unicode = MagicMock(return_value="test.iso")
+    dl_state = MagicMock(
+        get_infohash = MagicMock(return_value=b'aaaa'),
+        get_status = MagicMock(return_value=DownloadStatus.SEEDING),
+        get_download = MagicMock(return_value=fake_download),
+    )
```

Comment:
Please, fix the formatting:

```suggestion
        config=MagicMock(
            get_hops=MagicMock(return_value=0),
            get_safe_seeding=MagicMock(return_value=True),
        ),
    )
    fake_download.get_def().get_name_as_unicode = MagicMock(return_value="test.iso")
    dl_state = MagicMock(
        get_infohash=MagicMock(return_value=b'aaaa'),
        get_status=MagicMock(return_value=DownloadStatus.SEEDING),
        get_download=MagicMock(return_value=fake_download),
    )
```